### ISSUE TITLE
vtkSynchronizableRenderWindow can leak memory

### ISSUE THREAD

**Issue by Antti-Palola** [Kitware/vtk-js#2744]
### High-level description

Calling `vtkSynchronizableRenderWindow.getSynchronizerContext();` creates context if it didn't exist which is saved into a module var object. This is never freed causing a memory leak.

### Steps to reproduce

* Have a page with vtkJS and another without it
* Navigate to vtkJS page setting up stuff where `vtkSynchronizableRenderWindow.getSynchronizerContext();` is called
* Navigate back to the other page which should release vtkJS objects.

### Detailed behavior

https://github.com/Kitware/vtk-js/blob/517ac804bdaf177a552a51e903f50fd7b0ab0487/Sources/Rendering/Misc/SynchronizableRenderWindow/index.d.ts#L59

https://github.com/Kitware/vtk-js/blob/517ac804bdaf177a552a51e903f50fd7b0ab0487/Sources/Rendering/Misc/SynchronizableRenderWindow/index.js#L201-L212

### Expected behavior

A) There is a cleanup method in the module
B) The typing is changed so that a requested context can be nullified.

### Environment
- **vtk.js**: 26.8.0
- **OS**: MacOS 12.6.2
- **Browser**: Any latest browser

**Comment by Julesdevops** [comment]
a `setSynchronizerContext` function is exported from this module. Could we use it to free the synchronizer context?

**Comment by jourdain** [comment]
I agree with @Julesdevops here but we could also provide a simple `clearContexts()` if that helps?
